web/libs/editor/src/core/CustomTypes.ts

```diff
@@ -50,7 +50,30 @@ const CSSColor = types.custom<any, string>({
   },
 });
 
+// This type is using to store a raw callback function in mobx-state-tree model.
+// It won't be serialized right. However, it is useful to allow passing a callback function as a prop
+// and be able to get a reaction on its change
+//
+// /!\ Avoid using this type in case you need serialization of the model it contains
+const RawCallback = types.custom<Function, Function>({
```

Comment:
I think we definitely need to add more explanations on how and why would we use this new type, including:
- what was the high-level reason of adding new uncommon type to MST
- how would we create modals with this types from outside, examples maybe?
- what other considerations were there and why this one is the best option

Why I think that these are important questions they can be answered later.

Reply:
It was decided to take the solution of this issue beyond this PR